src/config.rs

```diff
@@ -713,6 +727,11 @@ impl Config {
     }
   }
 
+  fn format(ast: Module, search: &Search) -> Result<(), i32> {
+    fs::write(&search.justfile, ast.to_string()).unwrap();
```

Comment:
We should avoid the unwrap, since the error message won't be great if it fails. We can do something like:

```suggestion
  let Err(error) = fs::write(&search.justfile, ast.to_string()) {
    if self.verbosity.loud() {
      eprintln!(
        "Failed to write justfile to `{}`: {}",
        search.justfile.display(),
        error
      );
    }
  }
```